recipe/build_mamba.bat

```diff
@@ -10,6 +10,8 @@ rmdir /Q /S build
 mkdir build
 cd build
 
+set "CXXFLAGS=%CXXFLAGS% /D_LIBCPP_DISABLE_AVAILABILITY=1"
```

Comment:
I think this is only relevant on macOS, isn't it?

Reply:
well, can be. it doesn't hurt.  any on unix we define it unconditionally anyway too.  but sure, we can remove